srtp_cipher.go

```diff
@@ -10,11 +10,11 @@ import "github.com/pion/rtp"
 type srtpCipher interface {
 	// authTagLen returns auth key length of the cipher.
 	// See the note below.
-	rtpAuthTagLen() (int, error)
-	rtcpAuthTagLen() (int, error)
+	RtpAuthTagLen() (int, error)
+	RtcpAuthTagLen() (int, error)
 	// aeadAuthTagLen returns AEAD auth key length of the cipher.
 	// See the note below.
-	aeadAuthTagLen() (int, error)
+	AeadAuthTagLen() (int, error)
```

Comment:
Acronyms are all capitals in Go's naming convention. So `RTPAuth*`, `RTCPAuth*`, `AEADAuth*` would be preferred

Reply:
fixed